.github/actions/pr-to-update-go/pr_to_update_go/pr_template.md

```diff
@@ -0,0 +1,120 @@
+<!--
+************ STOP!! ************
+If this Pull Request is intended to fix a security vulnerability, DO NOT submit it! Instead, contact
+the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+-->
+## What does this PR (Pull Request) do?
+<!-- Explain the changes you made here. If this fixes an Issue, identify it by
+replacing the text in the checkbox item with the Issue number e.g.
+
+- [x] This PR fixes #9001 OR is not related to any Issue
+
+^ This will automatically close Issue number 9001 when the Pull Request is
+merged (The '#' is important).
+
+Be sure you check the box properly, see the "The following criteria are ALL
+met by this PR" section for details.
+-->
```

Comment:
I don't think you need comments like these, they're for humans to read

Reply:
Removed comments in de4b55e9f3